packages/wrapped-keys-lit-actions/project.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "wrapped-keys-lit-actions",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/wrapped-keys-lit-actions/src",
+  "projectType": "library",
+  "targets": {
+    "build-la": {
+      "cache": false,
+      "executor": "nx:run-commands",
+      "options": {
+        "commands": [
+          "yarn --cwd ./packages/wrapped-keys-lit-actions generate-lit-actions"
+        ],
+        "cwd": "."
+      }
+    },
+    "build": {
+      "cache": false,
+      "executor": "@nx/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/packages/wrapped-keys-lit-actions",
+        "main": "packages/wrapped-keys-lit-actions/src/index.ts",
+        "tsConfig": "packages/wrapped-keys-lit-actions/tsconfig.lib.json",
+        "assets": [
+          "packages/wrapped-keys-lit-actions/*.md",
+          {
+            "glob": "**/*",
+            "input": "packages/wrapped-keys-lit-actions/src/generated",
+            "output": "src/generated"
+          }
+        ],
+        "updateBuildableProjectDepsInPackageJson": true
+      },
+      "dependsOn": ["build-la"]
```

Comment:
This package could, in theory be completely bundled with esbuild instead of an esbuild script + typescript

I tried that for long and didn't manage to do it successfully in a clean way. Also this approach has some advantages:
- it generates typescript definitions
- it bundles each lit action independently and we can upload those to IPFS on future updates
- it maintains a similar build process than the other packages